lib/engine/game/g_1822/game.rb

```diff
@@ -953,6 +953,12 @@ def operating_round(round_num)
         end
 
         def payout_companies
+          # Functionality extracted so it can be over-ridden by subclasses
```

Comment:
I don't think having the rationale of why the logic was split out will be helpful to future developers. I suggest removing.

Reply:
I don't mind removing it, but I thought it would be better to explain why a method contained only a single call along with "super".

Reply:
My opinion is that its a common idiom used throughout the code base, so its not worth noting in this one place.